### PR TITLE
Adding missing feature descriptions - Batch 6

### PR DESCRIPTION
**Pull request by captainbrosset**

### CODE REVIEW

File: feature-group-definitions/nesting.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 name: Nesting
+description: For shorter selectors, easier reading, and more modularity, CSS rules can be nested inside other rules.
```
Comment (foolip) [inline feature-group-definitions/nesting.yml:2]:
```suggestion
description: CSS nesting allows for shorter selectors, easier reading, and more modularity by nesting rules inside others.
```

File: feature-group-definitions/offscreen-canvas.yml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 name: Offscreen canvas
+description: The `OffscreenCanvas` interface provides a canvas that can be drawn to off screen, with no dependencies on the DOM, which can be used to run heavy rendering operations inside a worker context.
```
Comment (foolip) [inline feature-group-definitions/offscreen-canvas.yml:2]:
```suggestion
description: The `OffscreenCanvas` API provides a canvas that can be drawn to off screen, with no dependencies on the DOM, which can be used to run heavy rendering operations inside a worker context.
```

File: feature-group-definitions/is.yml [hunk 3]
```diff
@@ -1,4 +1,5 @@
 name: ":is()"
+description: "The `:is()` CSS pseudo-class function takes a selector list as its argument, and selects any element that can be selected by one of the selectors in that list."
```
[the remark this answers is not in the document]
Comment (captainbrosset) [reply feature-group-definitions/is.yml:2]: See #862.

**Review by foolip — APPROVED** [review]

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
In the interest of getting https://github.com/web-platform-dx/web-features/pull/761 done ASAP I'm going to merge this and make a list of things that @ddbeck might want to review post facto and send tweaks for.

**Pull request merged**